src/java/org/eclipse/jetty/http2/hpack/Utils.java

```diff
@@ -0,0 +1,19 @@
+package org.eclipse.jetty.http2.hpack;
+
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.util.StringUtil;
+
+/**
+ * Helper class to simplify handling the jetty upgrade.
+ */
+public class Utils {
+    /**
+     * Copied from org.eclipse.jetty.http.HttpField#getLowerCaseName
+     */
+    static String getLowerCaseFieldName(final HttpField httpField) {
```

Comment:
Introduce helper method that copies code from `HttpField` which has a new method to retrieve the name as lower case.